release.sh

```diff
@@ -131,10 +131,19 @@ function master_version() {
   echo "${tokens[0]}.${tokens[1]}"
 }
 
+# Return the minor version of a release.
+# For example, "v0.2.1" returns "2"
+# Parameters: $1 - release version label.
+function minor_version() {
+  local release="${1//v/}"
+  local tokens=(${release//\./ })
```

Comment:
Can we keep the logic consistent with the `patch_version` function by splitting with `.`?